docs/core/upgrade.md

```diff
@@ -118,32 +152,17 @@ You MUST manually set the consensus version in the version map passed to the `Up
 ```go
 import foo "github.com/my/module/foo"
 
-app.UpgradeKeeper.SetUpgradeHandler("my-plan", func(ctx sdk.Context, plan upgradetypes.Plan, vm module.VersionMap)  (module.VersionMap, error) {
-  
+app.UpgradeKeeper.SetUpgradeHandler("my-plan", func(ctx sdk.Context, plan upgradetypes.Plan, fromVM module.VersionMap)  (module.VersionMap, error) {
+
     // Register the consensus version in the version map
     // to avoid the SDK from triggering the default
     // InitGenesis function.
-    vm["foo"] = foo.AppModule{}.ConsensusVersion()
+    fromVM["foo"] = foo.AppModule{}.ConsensusVersion()
 
     // Run custom InitGenesis for foo
     app.mm["foo"].InitGenesis(ctx, app.appCodec, myCustomGenesisState)
 
-    return app.mm.RunMigrations(ctx, cfg, vm)
-})
-```
-
-If you do not have a custom genesis function and want to skip the module's default genesis function, you can simply register the module with the version map in the `UpgradeHandler` as shown in the example:
```

Comment:
Why is this removed? I think it's still useful, no?

Reply:
This is moved / merged into "Genesis State" section: https://github.com/cosmos/cosmos-sdk/pull/10608/commits/37f75ce70df58af12ec03d2b9328daa9c16d9669#diff-7919f886ac1c280f5bc85e12992892df7f160c4ab535e878d2f1510f6d11489cR114

Reply:
I found it more logical to describe handling genesis state in one place. So I reordered the sections a bit.

Reply:
Ah I see , makes sense